charts/stable/wordpress/questions.yaml

```diff
@@ -102,7 +102,7 @@ questions:
                   type: string
                   private: true
                   default: ""
-  - variable: php-config
```

Comment:
This might break existing installations. Is there a reason for that change?

Reply:
```Error:  templates/: parse error at (wordpress/templates/_env.tpl:34): bad character U+002D '-'```

Reply:
Yea helm needs special treatment on that.